services/app-api/utils/validation/schemaMap.ts

```diff
@@ -139,7 +139,7 @@ export const radioOptional = () => radio().notRequired();
 
 // DYNAMIC
 export const dynamic = () =>
-  array().min(1).of(text()).required(error.REQUIRED_GENERIC);
+  array().min(1).of(mixed()).required(error.REQUIRED_GENERIC);
```

Comment:
Not sure how this will end up, but the backend one might need to stay like this -- depends how we decide to modify the onSubmit handler.